app/assets/javascripts/statistics.js

```diff
@@ -119,11 +119,17 @@ var drawBalanceChart = function drawBalanceChart(period, element) {
     };
     var data = response.data;
     var chartData = google.visualization.arrayToDataTable(data);
-    var chart = new google.visualization.ComboChart(css_id);
+    var chart = null;
+    if (css_id.id == 'main-balance') {
+      chart = new google.visualization.ComboChart(css_id);
+    } else {
+      chart = new google.visualization.ColumnChart(css_id);
+    }
     var formatter = new google.visualization.NumberFormat(response.currency_format);
     formatter.format(chartData, 1);
     formatter.format(chartData, 2);
-    formatter.format(chartData, 3);
+    if (css_id.id == 'main-balance') { formatter.format(chartData, 3) };
```

Comment:
@yunixon we don't use conditions in one line in JS. Please check our code-conventions.